apps/web/src/app/pages/trade/bridge/_components/XTransferModal.tsx

```diff
@@ -90,13 +94,23 @@ function XTransferModal() {
 
   const gasChecker = useXCallGasChecker(direction.from);
 
+  const ethereumChainId = useChainId();
```

Comment:
use useAccount's chainId.

Reply:
it doesn't work, the value is undefinend